src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -114,6 +114,11 @@ internal IEnumerable<IKeyValueAdapter> Adapters
         /// </summary>
         internal FeatureFilterTracing FeatureFilterTracing { get; set; } = new FeatureFilterTracing();
 
+        /// <summary>
+        /// Options used to configure provider startup.
+        /// </summary>
+        internal StartupOptions Startup { get; set; } = new StartupOptions();
```

Comment:
public?

Reply:
Not sure when I changed it to internal, will change it back